query/src/sessions/session_mgr.rs

```diff
@@ -90,6 +95,15 @@ impl SessionManager {
         let active_sessions = Arc::new(RwLock::new(HashMap::with_capacity(max_sessions)));
         let status = Arc::new(RwLock::new(Default::default()));
 
+        let (_guards, query_logger) = if conf.log.log_query_enabled {
+            let query_log_name = format!("query-detail-{}", conf.query.cluster_id,);
```

Comment:
what would be happened during reload when cluster id got changed.
I think file name could some thing not related to cluster id, tenant id, just relate to timestamp? since we already provide cluster and tenant inside each logging message